language/fr/permissions_ajax_shoutbox.php

```diff
@@ -35,7 +35,7 @@
 // in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
 //
 // Some characters you may want to copy&paste:
-// ’ » “ ” …
+// ’ « » “ ” …
```

Comment:
Ok, it is fine in that case. Just fix the whitespace for all files in that case.